src/openforms/emails/digest.py

```diff
@@ -179,3 +257,80 @@ def collect_broken_configurations() -> list[BrokenConfiguration]:
         )
 
     return broken_configurations
+
+
+def collect_invalid_registration_backends() -> list[InvalidRegistrationBackend]:
+    registration_backends = (
+        backend
+        for backend in FormRegistrationBackend.objects.select_related("form").iterator()
+        if backend.form.is_available
+    )
+
+    already_detected = []
+    invalid_registration_backends = []
+    for registration_backend in registration_backends:
+        form_name = registration_backend.form.admin_name
+        form = registration_backend.form
+        backend_options = registration_backend.options
+        backend_type = registration_backend.backend
+        plugin = register[backend_type]
+
+        # errors in general configuration are more straightforward for the user,
+        # so we show the exact ones
+        if plugin.verbose_name not in already_detected and (
+            broken_general_config := check_general_config(plugin)
+        ):
+            already_detected.append(plugin.verbose_name)
+            invalid_registration_backends.append(
+                InvalidRegistrationBackend(
+                    config_name=plugin.verbose_name,
+                    exception_message=broken_general_config,
+                    form_name=form_name,
+                )
+            )
+
+        # errors in the form level can be more detailed so here we want to show
+        # a more general error and the link to the broken form to investigate
+        if not is_form_config_valid(plugin, backend_options):
+            invalid_registration_backends.append(
+                InvalidRegistrationBackend(
+                    config_name=plugin.verbose_name,
+                    exception_message=_(
+                        "Invalid registration backend configuration has been detected at the form level"
+                    ),
+                    form_name=form_name,
+                    form_id=str(form.id),
+                    form_level=True,
+                )
+            )
+
+    return invalid_registration_backends
+
+
+def collect_invalid_logic_rules() -> list[InvalidLogicRule]:
+    forms = Form.objects.live()
+    static_variables = [var.key for var in get_static_variables()]
+
+    invalid_logic_rules = []
+    for form in forms:
+        form_variables_keys = [
+            form_variable.key for form_variable in form.formvariable_set.all()
+        ]
```

Comment:
I think you can get those more efficiently via a clever `.prefetch_related` call on the `forms` queryset.